### PR TITLE
add results section for scheduler experiments

### PR DESCRIPTION
**Pull request by deep-diver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,54 @@ image = pipeline(prompt, num_inference_steps=50).images[0]
 
 Follow [this guide](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv) to know more. 
 
+
+### Experimental results through various scheduler settings:
+
+<details>
+<summary>
+We have converted fine-tuned checkpoint for the dog images into Diffusers compatible StableDiffusionPipeline and ran various experiments with different scheduler settings. For example, the following parameters of the `DDIMScheduler` are tested on a different set of `guidance_scale` and `num_inference_steps`.
+</summary>
```
Comment (sayakpaul) [inline README.md:198]: The code quotes are not rendering:

image

File: README.md [hunk 2]
```diff
@@ -189,6 +189,54 @@ image = pipeline(prompt, num_inference_steps=50).images[0]
...
+      "prediction_type": [
+          "epsilon", 
+          "sample", 
+          "v_prediction"
+      ]
+  }
+}
+```
+
+Below shows the comparison between different values of `beta_schedule` parameters while others fixed as default. Take a look at [the original report](https://docs.google.com/spreadsheets/d/1_NhWuORn5ByEnvD9T3X4sHUnz_GR8uEtbE5HbI98hOM/edit?usp=sharing) which includes the results from other schedulers such as `PNDMScheduler` and `LMSDiscrete`.
```
Comment (sayakpaul) [inline README.md:230]:
```suggestion
Below is the comparison between different values of `beta_schedule` parameters while others are fixed to their default values. Take a look at [the original report](https://docs.google.com/spreadsheets/d/1_NhWuORn5ByEnvD9T3X4sHUnz_GR8uEtbE5HbI98hOM/edit?usp=sharing) which includes the results from other schedulers such as `PNDMScheduler` and `LMSDiscreteScheduler`. 
```

File: README.md [hunk 3]
```diff
@@ -189,6 +189,54 @@ image = pipeline(prompt, num_inference_steps=50).images[0]
...
+          "sample", 
+          "v_prediction"
+      ]
+  }
+}
+```
+
+Below shows the comparison between different values of `beta_schedule` parameters while others fixed as default. Take a look at [the original report](https://docs.google.com/spreadsheets/d/1_NhWuORn5ByEnvD9T3X4sHUnz_GR8uEtbE5HbI98hOM/edit?usp=sharing) which includes the results from other schedulers such as `PNDMScheduler` and `LMSDiscrete`. 
+
+It is often observed the default settings doesn't guarantee to generate better quality images. For example, the default values `guidance_scale` and `beta_schedule` are set to 7.5 and `linear`. However, when `guidance_scale` is set to 7.5, `scaled_linear` of the `beta_schedule` seems to work better. Or, when `beta_schedule` is set to `linear`, higher `guidance_scale` seems to work better.
```
Comment (sayakpaul) [inline README.md:232]:
```suggestion
It is often observed the default settings do guarantee to generate better quality images. For example, the default values of `guidance_scale` and `beta_schedule` are set to 7.5 and `linear`. However, when `guidance_scale` is set to 7.5, `scaled_linear` of the `beta_schedule` seems to work better. Or, when `beta_schedule` is set to `linear`, higher `guidance_scale` seems to work better. 
```

File: README.md [hunk 4]
```diff
@@ -189,6 +189,54 @@ image = pipeline(prompt, num_inference_steps=50).images[0]
...
+
+Below shows the comparison between different values of `beta_schedule` parameters while others fixed as default. Take a look at [the original report](https://docs.google.com/spreadsheets/d/1_NhWuORn5ByEnvD9T3X4sHUnz_GR8uEtbE5HbI98hOM/edit?usp=sharing) which includes the results from other schedulers such as `PNDMScheduler` and `LMSDiscrete`. 
+
+It is often observed the default settings doesn't guarantee to generate better quality images. For example, the default values `guidance_scale` and `beta_schedule` are set to 7.5 and `linear`. However, when `guidance_scale` is set to 7.5, `scaled_linear` of the `beta_schedule` seems to work better. Or, when `beta_schedule` is set to `linear`, higher `guidance_scale` seems to work better. 
+
+![](https://i.postimg.cc/QsW-CKTcv/DDIMScheduler.png)
+
+We have also run 4,800 experiments which generated 38,400 images in total. Those experiments are logged in Weights and Biases, and they are open to public. If you are curious, do check them out [here](https://wandb.ai/chansung18/SD-Scheduler-Explore?workspace=user-chansung18) as well as the [script](https://gist.github.com/deep-diver/0a2deb2cd369ab8c1bf3ee12f47d272a) that run the experiments.
```
Comment (sayakpaul) [inline README.md:236]:
```suggestion
We ran 4,800 experiments which generated 38,400 images in total. Those experiments are logged in Weights and Biases. If you are curious, do check them out [here](https://wandb.ai/chansung18/SD-Scheduler-Explore?workspace=user-chansung18) as well as the [script](https://gist.github.com/deep-diver/0a2deb2cd369ab8c1bf3ee12f47d272a) that was used to run the experiments. 
```

**Review by sayakpaul — APPROVED** [review]
Fantastic work! 

I think we don't need to put this in a summary block. We would want people to see the results right off the bat.

**Comment by deep-diver** [comment]
addressed comments. Thanks for the review!

**Pull request merged**